aggregator/src/binary_utils.rs

```diff
@@ -332,11 +332,14 @@ where
 /// body and status code 200. Each Janus component exposes this HTTP server to enable health
 /// checks, and to indicate when it has successfully started up.
 async fn health_endpoint_server(address: SocketAddr) {
-    let filter = warp::path("healthz")
-        .and(warp::get().or(warp::head()).unify())
-        .map(|| warp::reply::with_status(warp::reply::reply(), StatusCode::OK));
-    let server = warp::serve(filter);
-    server.bind(address).await;
+    let router = Router::new().get("/healthz", trillium::Status::Ok);
+    let handler = (Head::new(), router);
+    trillium_tokio::config()
+        .with_port(address.port())
```

Comment:
I should probably add a `with_socketaddr` that handles this port/host splitting, this seems like common behavior